cmd/kubexit/main.go

```diff
@@ -24,8 +26,13 @@ import (
 func main() {
 	var err error
 
-	// remove log timestamp
-	log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
+	if silence, _ := strconv.ParseBool(os.Getenv("KUBEXIT_SILENCE")); silence {
```

Comment:
Long-term: document these env vars and explain why they're needed in the README.